pandas/io/formats/format.py

```diff
@@ -945,19 +927,22 @@ def get_col_type(dtype):
                             compat.string_types):  # pragma: no cover
             raise AssertionError('column_format must be str or unicode, '
                                  'not {typ}'.format(typ=type(column_format)))
+        return column_format
 
-        if not self.longtable:
-            buf.write('\\begin{{tabular}}{{{fmt}}}\n'
-                      .format(fmt=column_format))
-            buf.write('\\toprule\n')
-        else:
-            buf.write('\\begin{{longtable}}{{{fmt}}}\n'
-                      .format(fmt=column_format))
-            buf.write('\\toprule\n')
+    def write_result(self, buf):
```

Comment:
I did some refactoring here and broke this method down into a couple of smaller ones. Ideally, pieces like `_rebuild_multi_index()` could also be a bit clearer, but maybe I'll tackle that while looking at some of the other Latex-related bugs.